cms/djangoapps/contentstore/utils.py

```diff
@@ -455,3 +457,10 @@ def get_visibility_partition_info(xblock):
         "has_selected_groups": has_selected_groups,
         "selected_verified_partition_id": selected_verified_partition_id,
     }
+
+
+def is_self_paced(course):
+    """
+    Returns True if course is self-paced, False otherwise.
+    """
+    return course and course.self_paced and SelfPacedConfiguration.current().enabled
```

Comment:
Can you check if there is any table in mysql where this mapping is added? I mean where we add information about course if it is self paced?